src/components/CustomerQuota/CheckoutSuccess/checkoutMessages.ts

```diff
@@ -17,7 +17,7 @@ export const getCheckoutMessages = (productType?: string): CheckoutMessage => {
       return {
         title: "Purchased!",
         description: "The following have been purchased:",
-        ctaButtonText: "Next customer"
+        ctaButtonText: "Next identity"
```

Comment:
Wondering if we should take out this property instead, since both product types result with the same text, what do you think?